Library/Formula/lorem.rb

```diff
@@ -1,10 +1,10 @@
 require 'formula'
 
 class Lorem < Formula
-  homepage 'http://code.google.com/p/lorem/'
-  url 'http://lorem.googlecode.com/svn-history/r4/trunk/lorem', :using => :curl
-  version '0.6.1'
-  sha1 'aa6ef66e5ee1151397f19b358d772af316cf333b'
+  homepage 'https://github.com/per9000/lorem'
+  url 'https://raw.githubusercontent.com/per9000/lorem/16239cd038d78956230efb40ccafe7fb0580b509/lorem'
```

Comment:
We'll need to stick to a stable release. We can apply an upstream commit as a patch if it's suitably essential.